Create My Reservations Page

KIsmoilov opened the pull request:
## Summary

- Created My Reservations page by following the required design

## Changes log
- [x] Created cards for each car on my reservations page.
- [x] Each car includes `car image`, `car name`, `daily price`, `total price`, `starting date`, `ending date`, `reserved label`


## Acceptance criteria

- [x] It should contain cards
- [x] For each card, it has: `car image`, `car name`, `daily price`, `total price`, `starting date`, `ending date`, `reserved label`

![Screen Shot 2023-04-04 at 4 53 35 PM (2)](https://user-images.githubusercontent.com/101260188/229783660-9c4d65f5-c5b6-4ee7-888b-00f5656e1b0f.png)

wassimchakib commented on src/assets/images/Camry.jpeg:
[Optional] Try to compress the file since it is large (2.63MB)

wassimchakib commented on src/components/Navbar/index.css:
```diff
@@ -1,5 +1,5 @@
 .navbar {
-  height: 100vh;
+  height: 100%;
```
Please make sure that the navbar has the minimum height of 100vh. For instance, this is how it looks like if there is no content
![image](https://user-images.githubusercontent.com/50754458/229946149-8a7ac5de-311e-4b7c-aadd-ea082b1e137a.png). By adding a `min-height`: 100vh, you will make sure that with or without the content, the copyright text will be at the end of the page

wassimchakib commented on src/components/my-reservations/MyReservations.css:
```diff
@@ -0,0 +1,106 @@
+  padding-top: 10px;
+}
+
+.reservation-card {
+  display: flex;
+  flex-direction: column;
+  background-color: rgb(239 243 248);
+  width: 300px;
+  height: 400px;
+  overflow: scroll;
+  border-top-left-radius: 50px;
+  border-bottom-left-radius: 20px;
+  border-bottom-right-radius: 20px;
```
Please make sure to make the card dynamic, don't fix the width and height, as it can have some unexpected behaviors: 
![image](https://user-images.githubusercontent.com/50754458/229946586-4de63a45-2921-4e25-be63-2b6327cf0408.png)
As you can see in the second card, the description overlaps the price because the height was fixed and it cant find the remaining space. Also, it's better to not have a scroll behavior for each card as it is against user experience.

wassimchakib reviewed (changes requested):
Hi @KIsmoilov , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![invalid-1](https://user-images.githubusercontent.com/50754458/229946912-fb9a8e02-c720-4caa-b0a4-fad0e20177b8.gif)

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit messages ✔️ 
- [ ] Used correct Gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Kweeka1 commented:
- [Optional] You could add pagination at the bottom when the items' count is more than 6.

KIsmoilov commented:
Hello @wassimchakib, @Kweeka1 

Thanks for your reviews. 

- Deleted the assets folder as it was not needed.
- Disabled the overflow to the cards.
- Changed the card's height to auto.
- Added min-height 100vh to the navbar.
- Implemented media queries to the width of the cards.

wassimchakib reviewed (approved):
Hi @KIsmoilov , 
I'm Wassim your reviewer for this PR. :wave: :wave: 

Thank you for implementing the changes, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media3.giphy.com/media/YKdm91XONLTaw/giphy-downsized-medium.gif"/>

## Highlights
:heavy_check_mark: Followed GitFlow.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Good PR summary.

## Optional suggestions
[OPTIONAL] N/A 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
